docs/guide/topics/audit_logs.rst

```diff
@@ -0,0 +1,131 @@
+
+Getting Audit Logs
+~~~~~~~~~~~~~~~~~~~~
+
+Audit logs can be retrieved via :func:`guild.audit_logs`, assuming you have the :attr:`~Permissions.view_audit_log` permission. 
+
+Note that this function returns an :class:`AsyncIterator` and so to properly go through the audit logs, you will need to iterate over them.
+
+Examples
+==========
+
+A handful of examples follow that walk through some use cases for looking at audit logs.
+
+Getting and displaying all audit log entries
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python3
+
+    async for entry in guild.audit_logs():
+        print(f'{entry.user} did {entry.action} to {entry.target}')
+
+Getting a specific number of entries
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python3
+
+    async for entry in guild.audit_logs(limit=50):
+        print(f'{entry.user} did {entry.action} to {entry.target}')
+
+Getting and displaying all actions of a certain type
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We may want to know how many times users have been kicked to look for certain users being kicked repeatedly instead of being banned:
+
+.. code-block:: python3
+
+    async for entry in guild.audit_logs(action=discord.AuditLogAction.kick):
+        print(f'{entry.user} kicked {entry.target}')
+
+Getting all entries made by a specific user
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Perhaps we wanted to check to see if a specific mod in our guild is possibly inactive. We could 
+achieve this by finding out how many actions they have performed in the past 90 days, like so:
+
+.. code-block:: python3
+
+    entries = [entry async for entry in guild.audit_logs(user=message.author)]
+    await channel.send(f'{message.author} has made {len(entries)} moderation actions in the last 90 days.')
+
+Sorting entries from oldest to newest
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python3
+    
+    async for entry in guild.audit_logs(oldest_first=True):
+        print(f'{entry.user} did {entry.action} to {entry.target}')
+
+Filtering to a specific time range
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This could be helpful if you are trying to find actions that occurred in a specific time frame. For 
+example, maybe we want to see how many messages were deleted in a specific timeframe:
+
+.. code-block:: python3
+
+    deletes = [entry async for entry in guild.audit_logs(before=message.id, after=othermessage.id, action=discord.AuditLogAction.message_delete)]
+    await channel.send(f'{len(deletes)} messages deleted before {discord.utils.format_dt(message.created_at)}')
+
+.. note:: 
+
+    If choosing to filter the audit log to entries after a specific time, the logs will be sorted from oldest to newest.
+
+You can find more information on the audit logs in the documentation for :meth:`~Guild.audit_logs`, :class:`AuditLogEntry`, :class:`AuditLogAction`, :class:`AuditLogActionCategory`, :class:`AuditLogChanges`, and :class:`AuditLogDiff`.
```

Comment:
A lot of references stacked into one sentence here. Maybe change this to a 'Further reading' section with a list?